app/controllers/api/authorizations_controller.rb

```diff
@@ -0,0 +1,20 @@
+# encoding: utf-8
+
+class Api::AuthorizationsController < Api::BaseController
+  def resource
+    Authorization.new(User.find(prx_auth_token.user_id))
+  end
+
+  def resources
+    Authorization.new(User.find(prx_auth_token.user_id)).accounts
+  end
+
+  def index_collection
```

Comment:
I am reconciling myself to this, but it seems odd to have accounts and authorization resources returned by the same controller.

I can't think of a better option really; maybe making a new controller just for this nested `accounts` index action - but that seems like it is not worth the effort.